contributors/guide/README.md

```diff
@@ -1,14 +1,3 @@
----
-title: "Contributors Guide"
-weight: 1
-slug: "guide" 
----
-
-<!-- 
-Contributing to this document? 
-Please use semantic line feeds for readability: http://rhodesmill.org/brandon/2012/one-sentence-per-line/ 
--->
-
```

Comment:
This page is no longer being included and our current practice is to word wrap at 80 characters following the [style guide](https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md)